### DISCUSSION TITLE
How to enable Named Entity Recognition

### DISCUSSION THREAD

**Discussion by fcscoutinho** [sepinf-inc/IPED#108] [Q&A] [answered]
[ERROR] [process.task.NamedEntityTask]                  edu/stanford/nlp/models/ner/english.all.3class.caseless.distsim.crf.ser.gz not found. Did you put the model in the optional lib folder?

(Machine translated) I don't know where to include the entity model for it to work with iped. iNCLUDE "stanford-corenlp-3.8.0.jar" in the optional_jars folder, but the "english.all.3class.caseless.distsim.crf.ser.gz" file that I downloaded I don't know what to do with it ...

**Comment by lfcnassif** [comment, marked as answer]
stanford-corenlp-3.8.0.jar is already distributed in optional_jars folder. You need to download the model (same version of the jar) from https://stanfordnlp.github.io/CoreNLP/history.html and put the model file(s) in the same folder.

**Comment by lfcnassif** [comment]
Did it work?

**Comment by fcscoutinho** [comment]
Yes.
Tank you!!!!

Em ter, 9 de jun de 2020 15:40, Luis Nassif <notifications@github.com>
escreveu:
